Index: tzdata-2020d/asia
===================================================================
--- tzdata-2020d.orig/asia
+++ tzdata-2020d/asia
@@ -672,8 +672,8 @@ Zone	Asia/Shanghai	8:05:43	-	LMT	1901
 			8:00	PRC	C%sT
 # Xinjiang time, used by many in western China; represented by Ürümqi / Ürümchi
 # / Wulumuqi.  (Please use Asia/Shanghai if you prefer Beijing time.)
-Zone	Asia/Urumqi	5:50:20	-	LMT	1928
-			6:00	-	+06
+#Zone	Asia/Urumqi	5:50:20	-	LMT	1928
+#			6:00	-	+06
 
 
 # Hong Kong
Index: tzdata-2020d/backward
===================================================================
--- tzdata-2020d.orig/backward
+++ tzdata-2020d/backward
@@ -33,6 +33,12 @@ Link	Pacific/Auckland	Antarctica/South_P
 Link	Asia/Ashgabat		Asia/Ashkhabad
 Link	Asia/Kolkata		Asia/Calcutta
 Link	Asia/Shanghai		Asia/Chongqing
+Link   Asia/Shanghai           Asia/Beijing
+Link   Asia/Shanghai           Asia/Chengdu
+Link   Asia/Shanghai           Asia/Nanjing
+Link   Asia/Shanghai           Asia/Wuhan
+Link   Asia/Shanghai           Asia/Xian
+Link   Asia/Shanghai           Asia/Urumqi
 Link	Asia/Shanghai		Asia/Chungking
 Link	Asia/Dhaka		Asia/Dacca
 Link	Asia/Shanghai		Asia/Harbin
