diff --git a/zone.tab b/zone.tab
index 9372c32..22c581f 100644
--- a/zone.tab
+++ b/zone.tab
@@ -145,14 +145,14 @@ CL	-3327-07040	America/Santiago	Chile (most areas)
 CL	-5309-07055	America/Punta_Arenas	Region of Magallanes
 CL	-2709-10926	Pacific/Easter	Easter Island
 CM	+0403+00942	Africa/Douala
-CN      +3114+12128     Asia/Shanghai    Beijing Time
-CN      +3992+11646     Asia/Beijing     Beijing Time
-CN      +3067+10406     Asia/Chengdu     Beijing Time
-CN      +2959+10654     Asia/Chongqing   Beijing Time
-CN      +3204+11878     Asia/Nanjing     Beijing Time
-CN      +3052+11431     Asia/Wuhan       Beijing Time
-CN      +3427+10895     Asia/Xian        Beijing Time
-CN      +4377+08768     Asia/Urumqi      Beijing Time
+CN	+3114+12128	Asia/Shanghai	Beijing Time
+CN	+3992+11646	Asia/Beijing	Beijing Time
+CN	+3067+10406	Asia/Chengdu	Beijing Time
+CN	+2959+10654	Asia/Chongqing	Beijing Time
+CN	+3204+11878	Asia/Nanjing	Beijing Time
+CN	+3052+11431	Asia/Wuhan	Beijing Time
+CN	+3427+10895	Asia/Xian	Beijing Time
+CN	+4377+08768	Asia/Urumqi	Beijing Time
 CO	+0436-07405	America/Bogota
 CR	+0956-08405	America/Costa_Rica
 CU	+2308-08222	America/Havana
diff --git a/zone1970.tab b/zone1970.tab
index e52945a..1fa11bf 100644
--- a/zone1970.tab
+++ b/zone1970.tab
@@ -137,14 +137,14 @@ CK	-2114-15946	Pacific/Rarotonga
 CL	-3327-07040	America/Santiago	Chile (most areas)
 CL	-5309-07055	America/Punta_Arenas	Region of Magallanes
 CL	-2709-10926	Pacific/Easter	Easter Island
-CN      +3114+12128     Asia/Shanghai    Beijing Time
-CN      +3992+11646     Asia/Beijing     Beijing Time
-CN      +3067+10406     Asia/Chengdu     Beijing Time
-CN      +2959+10654     Asia/Chongqing   Beijing Time
-CN      +3204+11878     Asia/Nanjing     Beijing Time
-CN      +3052+11431     Asia/Wuhan       Beijing Time
-CN      +3427+10895     Asia/Xian        Beijing Time
-CN      +4377+08768     Asia/Urumqi      Beijing Time
+CN	+3114+12128	Asia/Shanghai	Beijing Time
+CN	+3992+11646	Asia/Beijing	Beijing Time
+CN	+3067+10406	Asia/Chengdu	Beijing Time
+CN	+2959+10654	Asia/Chongqing	Beijing Time
+CN	+3204+11878	Asia/Nanjing	Beijing Time
+CN	+3052+11431	Asia/Wuhan	Beijing Time
+CN	+3427+10895	Asia/Xian	Beijing Time
+CN	+4377+08768	Asia/Urumqi	Beijing Time
 CO	+0436-07405	America/Bogota
 CR	+0956-08405	America/Costa_Rica
 CU	+2308-08222	America/Havana
